linera-summary/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "linera-summary"
+version = "0.1.0"
+description = "Executable for performance summary generation."
+readme = "README.md"
+publish = false
+
+authors.workspace = true
+repository.workspace = true
+homepage.workspace = true
+license.workspace = true
+edition.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+clap.workspace = true
+git2 = "0.19.0"
+linera-base.workspace = true
+linera-version.workspace = true
+octocrab = "0.42.0"
```

I think it would be better to move all imports to the root `Cargo.toml` file. Otherwise we risk adding duplicate dependencies with different versions :/

```suggestion
git2.workspace = true
linera-base.workspace = true
linera-version.workspace = true
octocrab.workspace = true
```